Dockerfile.ubuntu.18.04

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2019, Intel Corporation
+#
+# 'recipe' for Docker to build an Debian package
+#
+# Pull base image
+FROM ubuntu:18.04
+Maintainer John.E.Malmberg <John.E.Malmberg@intel.com>
+
+# Install basic tools
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+            autoconf bash curl debmake debhelper dpkg-dev doxygen gcc \
+            git locales make patch rpm
+
+# FUSE3
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+            pkg-config udev python3 python3-pytest
+
+# Fuse 3 need meson > 0.46 so pull from Ubuntu 18.10
```

Comment:
There is a [project](https://github.com/daos-stack/meson) for this already.

Reply:
The Ubuntu meson package does not contain any architecture specific binaries.

Until we get a local Ubuntu repository setup, no advantage adding Ubuntu 18.04 to that [project](https://github.com/daos-stack/meson)

Reply:
I'm not sure I'm understanding the relevance of a Ubuntu repository in regards to adding Ubunto to our existing meson project.  At least with the latter you have a repository you can add and then let the repository tools do the work instead of specifying URLs directly.

Reply:
Currently we can not use Jenkins as a Debian/Ubuntu repository, so instead of using an Ubuntu URL, I would have to specify a Jenkins URL to pull essentially the exact same package.

There is no way currently to let the repository tools do the work.  That is going to be a later step.